examples/analog/AI_multipleChannels.py

```diff
@@ -19,11 +19,11 @@
    AI2: AI2 to AI6
    AI3: AI3 to AI7
 
-A time gap exists between the end of one signal acquisition and the start of
-the next signal acquisition when you specify N samples (reading multiple
-points) mode for Analog Input API. Refer to the following link for more
-information about the time gap.
-https://github.com/ni/NI-ELVIS-III-Python-Examples/blob/master/docs/NI_ELVIS_III_Understanding_Gaps.md
+The Python ELVIS III Analog Input API provides a function for you to perform
```

Comment:
1. `APIs provide` because there are more one APIs
2. Ask Ling to review the sentence.